docs/discover/view-single-document.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[discover-view-single-document]]
+== View single document
+
+Once you've found a document of interest in Discover there's a simple way to share or bookmark it. 
+
+. In the document table, click the expand icon (>).
+. Click **View single document** 
++
+This allows you to view the document in two ways. **Table** displays the document's fields row by row.
+**JSON** (JavaScript Object Notation) allows you to take a look at how the document is returned from Elasticsearch.
++
+[role="screenshot"]
+image::images/discover-view-single-document.png[Discover single document view]
+
+The link is valid for the time the document is available in Elasticsearch. If you need another way to view single 
```

Comment:
What link?

The last sentence sounds like it's getting down in the weeds.  Is it needed here?

Reply:
true, thanks removed it